src/validate.ts

```diff
@@ -26,31 +24,36 @@ const validateUploader = async (body) => {
   return true;
 };
 
-export const retrieveChecksum = async (version, encryption) => {
-  const url = `https://raw.githubusercontent.com/codecov/codecov-bash/${version}/SHA${encryption}SUM`;
```

Comment:
Please note the previous implementation has a flaw, as the `version` may contain `..` (parent directory), so it can point to another repo.